src/App.tsx

```diff
@@ -1,16 +1,58 @@
-import React from "react";
+import React, { useState } from "react";
 import "./App.css";
 import Logo from "./logo.png";
+import { ValidateEmail, ValidatePassword } from "./utils/validate";
 
 function App() {
+  const [email, setEmail] = useState(" ");
+  const [password, setPassword] = useState(" ");
+  const [errorEmail, setErrorEmail] = useState("");
+  const [errorPassword, setErrorPassword] = useState("");
+
+  const handleSubmit = (e: React.FormEvent<HTMLFormElement>) => {
+    ValidateEmail(email) ? setErrorEmail("E-mail Inválido") : setErrorEmail("");
+
+    ValidatePassword(password)
+      ? setErrorPassword(
+          "Password Inválido Lembre-se: Mínimo 7 caracteres, deve conter, ao menos ,uma letra e um número."
+        )
+      : setErrorPassword("");
```

Comment:
Eu sugeriria usar ternários mais para condições curtas e que são retornadas ou assinaladas a alguma variável. No caso de execução condicional de funções void, o bom e velho if é sempre mais claro e legível. O que acha?

Outro ponto é que ternários são um pouco mais difíceis de ler quando quebram linha.

Reply:
exemplificando a sugestão do @alanraso:
```suggestion
   if (isValidEmail(email)) {
     setErrorEmail("")
   } else {
     setErrorEmail("E-mail Inválido")
   }
   
   if (isPasswordValid(password)) {
     setErrorPassword("");
   } else {
    setErrorPassword(
          "Password Inválido Lembre-se: Mínimo 7 caracteres, deve conter, ao menos ,uma letra e um número."
        ); 
   }
```
obs: funções são nomeadas com letra minúscula. e, uma prática comum é nomear funções de validação  como se fossem perguntas. então, ao invés de ValidateEmail seria isEmailValid 👍